CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - The GET `/opportunities` endpoint now returns an `OpportunityBundle` and accepts pagination parameters.
+- The `Organization` attribute `taxId` has been renamed to `taxId`.
```

Comment:
```suggestion
- The `Organization` attribute `employerIdentificationNumber` has been renamed to `taxId`.
```

Reply:
Sorry yeah that was the *second* change.

Reply:
😐